cli/src/semgrep/dependency_aware_rule.py

```diff
@@ -54,61 +55,33 @@ def parse_depends_on_yaml(
         )
 
 
-def run_dependency_aware_rule(
-    matches: List[RuleMatch],
-    rule: Rule,
-    dep_trie: DependencyTrie,
-) -> Tuple[List[RuleMatch], List[SemgrepError]]:
-    """
-    Run a dependency aware rule. These rules filters the results based on the precense or absence
-    of dependencies. Dependencies are determined by searching for lockfiles under the first entry
-    in the `targets` argument.
-    """
+def generate_non_reachable_sca_findings(
```

Comment:
```suggestion
def generate_unreachable_sca_findings(
```

optionally we could make this consistent with phrasing elsewhere